<launch>
  <node pkg="pr2_mechanism_controllers" type="send_periodic_cmd_srv.py"
        name="laser_cmd"
        args="laser_tilt_controller linear 10 .5 .35" />

  <node type="laser_scan_assembler" pkg="laser_assembler"
        name="tilt_scan_assembler">
    <remap from="scan" to="tilt_scan"/>
    <param name="max_scans" type="int" value="400" />
    <param name="fixed_frame" type="string" value="base_link" />
    <param name="ignore_laser_skew" type="bool" value="true" />
  </node>

  <node pkg="pr2_arm_navigation_perception" type="pr2_laser_snapshotter" output="screen"
        name="snapshotter">
    <remap from="laser_scanner_signal"
           to="/laser_tilt_controller/laser_scanner_signal"/>
    <remap from="full_cloud" to="tilt_laser_cloud" />
  </node>
</launch>
